packages/components/src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,217 @@
+     */
+    children,
+    /**
+     * additional classed for custom styling
+     */
+    className,
+    /**
+     * Whether or not an accordion item is disabled from user interaction.
+     *
+     * @default false
+     */
+    disabled,
+    /**
+     * click callback for this item
+     */
+    onClick,
+    /**
+     * A string value for the accordion item. All items within an accordion should use a unique value.
+     */
+    value,
+}: AccordionItemProps) => {
+    return (
+        <RadixAccordion.Item
+            className={cn([styles.accordionItem, className])}
+            value={value}
+            onPointerDown={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'false';
+            }}
+            onBlur={(event) => {
+                event.currentTarget.dataset.showFocusRing = 'true';
+            }}
+            onClick={onClick}
+            disabled={disabled}
+            data-test-id={dataTestId}
+        >
+            {children}
+        </RadixAccordion.Item>
+    );
+};
+AccordionItem.displayName = 'Accordion.Item';
+
+export type AccordionHeaderProps = {
+    /**
+     * Children of the Accordion header. This should contain `Accordion.Trigger`
+     */
+    children?: ReactNode;
+    /**
+     * additional classed for custom styling
+     */
+    className?: string;
+};
+
+export const AccordionHeader = ({ children, className }: AccordionHeaderProps) => {
+    return <RadixAccordion.Header className={cn([className])}>{children}</RadixAccordion.Header>;
+};
+AccordionHeader.displayName = 'Accordion.Header';
+
+export type AccordionTriggerProps = { children?: ReactNode; className?: string; 'data-test-id'?: string };
+
+export const AccordionTrigger = ({
+    'data-test-id': dataTestId = 'fondue-accordion-trigger',
+    /**
+     * Children of the Accordion trigger. This contains the actually clickable and visible header content
+     */
+    children,
+    /**
+     * additional classed for custom styling
+     */
+    className,
+}: AccordionTriggerProps) => {
+    return (
+        <RadixAccordion.Trigger className={cn([styles.accordionTrigger, className])} data-test-id={dataTestId}>
+            {children}
+            <IconCaretDown className={styles.accordionCaret} size="16" />
+        </RadixAccordion.Trigger>
+    );
+};
+AccordionTrigger.displayName = 'Accordion.Trigger';
+
+type AccordionContentProps = {
+    'data-test-id'?: string;
+    /**
+     * Children of the Accordion content. This contains the main content
+     */
+    children?: ReactNode;
+    /**
+     * additional classed for custom styling
+     */
+    className?: string;
+    /**
+     * adds a divider line between the header and the content
+     */
+    divider?: boolean;
+    /**
+     * click callback for the content
+     */
+    onClick?: MouseEventHandler<HTMLDivElement>;
+    /**
+     * controls if we show paddings around the content
+     * @default treu
+     */
+    padding?: boolean;
```

Comment:
Instead of boolean, maybe the same as Divider? `'none' | 'small' | 'medium' | 'large'`